src/adapters/openai/helpers/embedding.ts

```diff
@@ -0,0 +1,25 @@
+import OpenAI from "openai";
+import { Context } from "../../../types";
+import { SuperOpenAi } from "./openai";
+const VECTOR_SIZE = 512;
```

Comment:
Why?

Reply:
Could be expanded to add support, for text generation and question answering. I think it should be kept as a separate adapter.